README.md

```diff
@@ -68,8 +68,15 @@ Please read the [*Contributing to CodeIgniter*](https://github.com/bcit-ci/CodeI
 ## Server Requirements
 PHP version 7.1 or higher is required, with the following extensions installed: 
 
-- intl
+- [intl](http://php.net/manual/en/intl.requirements.php)
+- [libcurl](http://php.net/manual/en/curl.requirements.php) if you plan to use the HTTP\CURLRequest library
```

Comment:
`ext/curl` can be added to `composer.json` in `suggest` part.

Reply:
Hmm - that package isn't found on packagist.org. Looking at the project homepage, it looks like they expect it to be built from source. It looks like an O/S level thing, rather than a PHP dependency.
I did find a curl/curl, which is an OO wrapper for the curl extension - that still doesn't address installing libcurl.

Reply:
it looks ike ext-curl would be the ticket, for a php extension :) adding that